chosen/chosen.php

```diff
@@ -10,9 +10,9 @@ public function register() {
 		
 		$this->name        = plugin_lang_get( 'title' );
 		$this->description = plugin_lang_get( 'description' );
-		$this->version     = '1.2.0';
+		$this->version     = '1.2.1';
 		$this->requires    = array(
-			'MantisCore' => '1.2.0, 1.3.0',
+			'MantisCore' => '2.0.0',
```

Comment:
We should add the older MantisBT version, that other can use the plugin also in older MantsBT version. Can you explain this also for the older version?